Dockerfile-ppa

```diff
@@ -3,11 +3,11 @@ MAINTAINER Grant Heffernan <grant@mapzen.com>
 
 ENV TERM vt100
 
-ENV DATA_DIR ${DATA_DIR:-"/data"}
+ENV DATA_DIR ${DATA_DIR:-"/data/valhalla"}
 ENV CONF_FILE ${CONF_FILE:-"/conf/valhalla.json"}
 
 RUN apt-get update -y && apt-get upgrade -y
-RUN apt-get install jq supervisor software-properties-common -y
+RUN apt-get install jq spatialite-bin wget unzip supervisor software-properties-common -y
```

Comment:
shouldnt the ppa not need these? in fact shouldnt it not need jq either. i thought the ppa knew about these deps and got them automatically